lib/src/cpp-test/array/serializer.cpp

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Can we make a gtest of it?

Reply:
Not strictly, it would require a pre executed python step to create the cereal binary file.